crowdsec-docs/unversioned/getting_started/installation/linux.mdx

```diff
@@ -0,0 +1,146 @@
+
+:::info
+We understand that security conscious users may prefer not to use `curl | bash` syntax, [manual install options here](https://packagecloud.io/crowdsec/crowdsec/install#manual)
+:::
+
+<Tabs
+  defaultValue="debian"
+  groupId="operating-systems"
+  values={[
+    {label: 'Debian/Ubuntu', value: 'debian'},
+    {label: 'EL/Centos7', value: 'centos7'},
+    {label: 'EL/Centos Stream 8', value: 'centos8'},
+    {label: 'Amzn Linux 2', value: 'amz'},
+    {label: 'OpenWRT', value: 'openwrt'},
+    {label: 'CloudLinux', value: 'cloudlinux'},
+  ]}>
+  <TabItem value="debian">
+    ```bash
+    curl -s https://packagecloud.io/install/repositories/crowdsec/crowdsec/script.deb.sh | sudo bash
+    ```
+  </TabItem>
+  
+  <TabItem value="centos7">
+    ```bash
+    curl -s https://packagecloud.io/install/repositories/crowdsec/crowdsec/script.rpm.sh | sudo bash
+    ```
+  </TabItem>
+
+  <TabItem value="centos8">
+    ```bash
+    curl -s https://packagecloud.io/install/repositories/crowdsec/crowdsec/script.rpm.sh | sudo bash
+    ```
+  </TabItem>
+
+  <TabItem value="amz">
+    ```bash
+    curl -s https://packagecloud.io/install/repositories/crowdsec/crowdsec/script.rpm.sh | sudo bash
+    ```
+  </TabItem>
+
+  <TabItem value="openwrt">
+    OpenWRT packages are available in the official repositories.
+  </TabItem>
+
+  <TabItem value="cloudlinux">
+    ```bash
+    curl -s https://packagecloud.io/install/repositories/crowdsec/crowdsec/script.rpm.sh | os=el dist=7 sudo bash
+    ```
+  </TabItem>
+</Tabs>
+
+### Install Security Engine
+
+Once the repository is added, you can install the security engine via:
+
+<Tabs
+  defaultValue="debian"
+  groupId="operating-systems"
+  values={[
+    {label: 'Debian/Ubuntu', value: 'debian'},
+    {label: 'EL/Centos7', value: 'centos7'},
+    {label: 'EL/Centos Stream 8', value: 'centos8'},
+    {label: 'Amzn Linux 2', value: 'amz'},
+    {label: 'OpenWRT', value: 'openwrt'},
+    {label: 'CloudLinux', value: 'cloudlinux'},
+  ]}>
+  <TabItem value="debian">
+    <CodeBlock className="language-bash">apt install crowdsec</CodeBlock>
+  </TabItem>
+  
+  <TabItem value="centos7">
+    <CodeBlock className="language-bash">yum install crowdsec</CodeBlock>
+  </TabItem>
+
+  <TabItem value="centos8">
+    <CodeBlock className="language-bash">dnf install crowdsec</CodeBlock>
+  </TabItem>
+
+  <TabItem value="amz">
+    <CodeBlock className="language-bash">yum install crowdsec</CodeBlock>
+  </TabItem>
+
+  <TabItem value="openwrt">
+    <CodeBlock className="language-bash">opkg install crowdsec</CodeBlock>
+  </TabItem>
+
+  <TabItem value="cloudlinux">
+    <CodeBlock className="language-bash">yum install crowdsec</CodeBlock>
+  </TabItem>
+</Tabs>
+
+
+### Install Remediation Component
+
+:::warning
+Security Engine by itself is a detection engine, it will not block anything. You need to install a Remediation Component to enforce decisions
```

Comment:
We should link "Remediation Component" to a definition somewhere. If the user is a newcomer, it might be confusing as it's a made-up term.